test/es7/immutable-prototype.js

```diff
@@ -0,0 +1,77 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+
+// ES7 Object Prototype object has an immutable [[Prototype]] internal slot
+// See: 19.1.3 Properties of the Object Prototype Object
+// See: 9.4.7 Immutable Prototype Exotic Objects
+
+WScript.LoadScriptFile("..\\UnitTestFramework\\UnitTestFramework.js");
+
+var tests = [
+    {
+        name: "Okay to set arbitrary object's [[Prototype]] using __proto__",
```

Comment:
Ah, I figured there would be a test already. Since I wanted to make sure the new behavior didn't break the old behavior I added these tests here, but I see that they are redundant so I'll remove them.